src/features/bulk/FileViewer/styles.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
```

Comment:
I'm not a fan of the decoupling between components and their styles. The convention laid out by ASC (which we follow by default as this comes from the frontend guild) dictates that styles should be placed in a document named after the component file name. This will make locating the styles through an IDE (using keyboard only) much simpler as well.

Reply:
split these into subfolders with individual style files.